src/main/resources/db/migration/V1__create_client_table.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE `client` (
+    id int AUTO_INCREMENT NOT NULL,
+    name nvarchar(50) NOT NULL,
+
+    # Address
```

Comment:
why is there a comment here, db field names should be self explanatory e.g. addressLine1, addressLine2 is a common standard along with the rest of the fields. We don't need this comment